libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -27,7 +29,12 @@ export class BookSearchComponent {
   constructor(
     private readonly store: Store,
     private readonly fb: FormBuilder
-  ) {}
+  ) {
+    this.searchTermSubject.pipe(distinctUntilChanged(), debounceTime(500), takeUntil(this.unsubscribeSubject)).subscribe(searchTerm => {
```

Comment:
Please use something like Prettier to format this.

Reply:
Updated changes